articles/cognitive-services/openai/how-to/chatgpt.md

```diff
@@ -25,12 +25,12 @@ The following code snippet shows the most basic way to use the ChatGPT model. We
 import os
 import openai
 openai.api_type = "azure"
-openai.api_base = "https://{your-resource-name}.openai.azure.com/"
+openai.api_base = "https://{your-resource-location}.api.cognitive.microsoft.com/"
 openai.api_version = "2022-12-01"
 openai.api_key = os.getenv("OPENAI_API_KEY")
 
 response = openai.Completion.create(
-  engine="gpt-35-turbo",
+  engine="{your-resource-name}",
```

Comment:
The use of gpt-35-turbo is very much intentional here to make clear you have to use the ChatGPT model, but it is really valuable feedback that this is confusing to people and we need to make explicit that ultimately this value will match whatever unique name you assign to your deployment for the gpt-35-turbo model. By convention in our examples we often mirror model name and deployment name, but we can definitely do a better job making this clear.